overlays/turnkey.d/protocol-hardening/etc/modprobe.d/rds.conf

```diff
@@ -0,0 +1 @@
+install rds bin/true
```

Comment:
*probably* doesn't matter if you've checked it, but this is a relative path, might be best to add the leading slash

Reply:
Good catch @OnGle

@Christie018 - could you please make that change and double check that it works as expected. Thanks.